ui/admin/tests/unit/controllers/application-test.js

```diff
@@ -58,4 +60,55 @@ module('Unit | Controller | application', function (hooks) {
 
     assert.true(featuresService.isEnabled('ssh-target'));
   });
+
+  test('customRouteChangeValidator returns true when transitioning between different routes', function (assert) {
+    const transition = {
+      from: router.recognize('/scopes/global/scopes'),
+      to: router.recognize('/scopes/global/auth-methods'),
+    };
+
+    assert.true(controller.customRouteChangeValidator(transition));
+
+    transition.from = undefined;
+
+    assert.true(controller.customRouteChangeValidator(transition));
+  });
+
+  test('customRouteChangeValidator returns true when transitioning between different routes with the same name', function (assert) {
```

Comment:
Would it be more accurate to say this is transitioning between same routes but different path parameters?

Reply:
yeah and thats clearer too. thank u, ill update the wording